operator/data/profiles/default.yaml

```diff
@@ -54,7 +54,7 @@ spec:
 
   # Policy feature
     policy:
-      enabled: true
+      enabled: false
```

Comment:
Assuming this controls the deployment itself, lgtm

@douglas-reid ?

Reply:
Yes, this is controls the deployment